src/components/ConnectForm.vue

```diff
@@ -435,6 +444,10 @@ export default {
 	padding: 2.5em 3em;
 	font-family: 'Ubuntu', sans-serif;
 }
+.connectContainer .helpBtn {
+	float: right;
+	cursor: pointer;
```

Comment:
I guess it makes sense to define this globally for all help buttons?

Reply:
You never know where you will want to place them, so I'd say no. Better go case-by-case.

Reply:
Oh wait, I thought this was "cursor: help;". Nevermind.